code/lambdas/CMakeLists.txt

```diff
@@ -3,17 +3,17 @@
 cmake_minimum_required( VERSION 3.1 )
 project( lambdas LANGUAGES CXX )
 
+# This example only works warning-free in C++11 mode with MSVC and Clang.
+set( CMAKE_CXX_STANDARD 11 )
+
```

Comment:
I usually prefer `target_compile_features(lambdas_randomize PUBLIC cxx_std_11)` since I am never sure how `CMAKE_CXX_STANDARD` interacts with different projects that are part of the same build. But up to you!

Reply:
There are arguments to be made for both approaches. When you write a library that requires a certain C\+\+ standard from its user (who uses it independently of you) to be able to interpret its public headers, then setting this compile feature can be very good.

But I found that when building for instance the ATLAS offline project, where we control the build of everything ourselves, `CMAKE_CXX_STANDARD` is just the way to go...